### PR TITLE
feat: make code dry

### PR DESCRIPTION
**Pull request by fairlighteth**
# Summary

- Use the same import for products card

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @fairlighteth , could you please clarify what in particular I should test here?

File: apps/cow-fi/data/home/const.tsx [hunk 1]
```diff
@@ -96,3 +111,56 @@ export const CHANNEL_LIST = [
     iconImage: IMG_SNAPSHOT,
   },
 ]
+
+export const PRODUCT_CONTAINERS = (
```
Comment (anxolin) [inline apps/cow-fi/data/home/const.tsx:115]: not sure if const is the place for this. Now it worked to not create a component cause you were able to build it from static information (no props needed), but not always be possible

I will approve, just to have it DRYer the code, but I think we will need some other general solution imo

**Review by anxolin — APPROVED** [review]

**Comment by fairlighteth** [comment]
> Hey @fairlighteth , could you please clarify what in particular I should test here? 

Homepage and /products page should work as expected

**Review by shoom3301 — APPROVED** [review]

**Comment by elena-zh** [comment]
Works as expected

**Pull request merged**